### DISCUSSION TITLE
Is 500 concurrent Realtime connections the max?

### DISCUSSION THREAD

**Discussion by nigyel** [supabase/supabase#18017] [Questions]
Hello all,

What is the maximum number of Concurrent Users in Supabase? (How many users can log-in at the same time?)

Thanks,
Nig.

**Comment by GaryAustin1** [comment]
Depends on your plan. https://supabase.com/docs/guides/realtime/quotas
https://supabase.com/pricing

Realtime connection is usually 1 per client with multiple channels per connection.
